web/libs/datamanager/src/utils/packJSON.js

```diff
@@ -1,3 +1,5 @@
+// DEPRECATED; don't use this. use utils/urlJSON instead.
```

Comment:
better to use jsdoc comment here
```suggestion
/** @deprecated Buggy legacy library, don't use it. Use utils/urlJSON instead. */
```